README.md

```diff
@@ -98,14 +98,31 @@ class MyFunction implements RequestHandler<String, String> {
     }
 ```
 
-### Support Java 11 and Above 
+### Support Java 11 and Above
 
 Add the environment variable `JAVA_TOOL_OPTIONS` to your Lambda functions and set it to
 `-Djdk.attach.allowAttachSelf=true` in addition to the manual code mentioned above.
 
 ### Supported Instrumentation Libraries
 
-- Aws SDK V1 
+- Aws SDK V1
 - Aws SDK V2
 - Apache HTTP Client
-- Apache Kafka
\ No newline at end of file
+- Apache Kafka
+
+### Secret scrubbing
+
+The tracer will automatically scrub values for keys matching the following regex patterns:
```

Comment:
Not sure I got the question. If the regex is `.*abc.*`, then it would match a key with `abc` in it like `aaabcaaa`, and if the regex is an exact match such as `xyz` it would match only keys named `xyz`, so it's really up to how the user will use it

Reply:
There is a difference between regex `match` and regex `search`
* does `abc` regex match `aaaaaaabcaaaaa` - no
* does `abc` regex search `aaaaaabcaaaaa` - yes (Could have better Grammer here ;) )

In other distros we confused customers with `search` and `match`, and it looks like you are talking about a complete regex match, from the start until the end of the string